client/hosting-overview/index.tsx

```diff
@@ -1,9 +1,35 @@
 import page from '@automattic/calypso-router';
 import { makeLayout, render as clientRender } from 'calypso/controller';
-import { siteSelection, sites } from 'calypso/my-sites/controller';
-import hostingOverview from './controller';
+import { navigation, siteSelection, sites } from 'calypso/my-sites/controller';
+import { redirectHomeIfIneligible } from 'calypso/my-sites/github-deployments/controller';
+import { handleHostingPanelRedirect } from 'calypso/my-sites/hosting/controller';
+import { hostingOverview, hostingConfiguration, hostingActivate } from './controller';
 
 export default function () {
 	page( '/hosting-overview', siteSelection, sites, makeLayout, clientRender );
 	page( '/hosting-overview/:site', siteSelection, hostingOverview, makeLayout, clientRender );
+
+	page( '/hosting-config', siteSelection, sites, makeLayout, clientRender );
```

Comment:
hmm should these also be removed from `client/my-sites/hosting/index.js`? they seem to be duplicated now

Reply:
No, they are still required to show the pages in Nav Unification (when the flag is not enabled).

Reply:
Sorry I think you're right; they seem to be unused now. We can clean up in a follow-up PR.